### PR TITLE
NXP-29026: fix the CSV export retention action

### PR DESCRIPTION
**Pull request by RSalem07**

### CODE REVIEW

File: nuxeo-retention-web/search/retention/nuxeo-retention-search-results.html [hunk 1]
```diff
@@ -22,15 +22,7 @@
 -->
 <dom-module id="nuxeo-retention-search-results">
   <template>
-    <style include="nuxeo-styles">
-      nuxeo-data-table,
```
Comment (nmpcunha) [inline nuxeo-retention-web/search/retention/nuxeo-retention-search-results.html:26]: Don't we need this styling, at least for `nuxeo-data-table`?

**Review by nmpcunha — APPROVED** [review]
Despite the comment on the styling removal, it looks good to me 👍🏼 
Guess it will be easier to validate the styling question with the preview (WIP)

**Review by andreacornaglia — APPROVED** [review]

File: nuxeo-retention-web/search/retention/nuxeo-retention-search-results.html (hunk 1 above)
Comment (RSalem07) [reply nuxeo-retention-web/search/retention/nuxeo-retention-search-results.html]: Yeah, this modification is not needed, it was copied past from my different tests. Thx

**Comment by nmpcunha** [comment]
LGTM 👍🏼

**Pull request merged**